schemas/PositiveFiniteAmount.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "PositiveFiniteAmount",
+  "title": "PositiveFiniteAmount",
+  "description": "A string representation of a positive, finite number",
+  "type": "string",
+  "pattern": "/^[+]?([1-9][0-9]*(?:[.][0-9]*)?|0*.0*[1-9][0-9]*)(?:[eE][+-][0-9]+)?$/"
```

Comment:
https://github.com/interledger/five-bells-shared/blob/master/schemas/Amount.json#L6 is much shorter, and I think you just need to change it to:

```
"^[+]?[1-9]*[.]?[0-9]+([eE][-+]?[0-9]+)?$"
```